### PR TITLE
Testing - Vera Simon

### PR DESCRIPTION
**Pull request by VeraSimon**
@bradiowave

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -1,3 +1,591 @@
 const helpers = require('./project-1');
 
 // start testing!
+
+describe('project-1.js', () => {
+	describe('multiplyByTen()', () => {
+		// const multiplyByTen = (num) => {
+		// 	return num * 10;
+		// };
+		// expected multiplication
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:10]: Great comments to show each function you are testing.

File: basic-javascript/project-1.spec.js [hunk 2]
```diff
@@ -1,3 +1,591 @@
...
+		// less than 90
+		it('should return true if input is less than ninety', () => {
+			const actual = helpers.lessThanNinety(42);
+			expect(actual).toEqual(true);
+		});
+
+		// greater than or equal to 90
+		it('should return false if input is greater than eighty-nine', () => {
+			const actual = helpers.lessThanNinety(90);
+			expect(actual).toEqual(false);
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:124]: Good edge case test to make sure the function does exactly what it says.

File: basic-javascript/project-1.spec.js [hunk 3]
```diff
@@ -1,3 +1,591 @@
...
+	});
+
+	describe('getGreeting()', () => {
+		// const getGreeting = (name) => {
+		// 	return `Hello ${name}!`;
+		// };
+		// expected greeting string
+		it('should return a string like "Hello Dolly!"', () => {
+			const actual = helpers.getGreeting('Vera');
+			expect(actual).toMatch(/^Hello Vera!$/);
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:489]: Cool tests using regex!

File: basic-javascript/project-2.spec.js [hunk 4]
```diff
@@ -1,3 +1,519 @@
...
+		// 	return arr;
+		// };
+		// arr length++ and arr.length-1 === item
+		it('should return the array passed with a new value at the end', () => {
+			const arr = [0, 1, 2, 3, 4];
+			const arrPlusOne = funcs.addItemToArray(arr, 5);
+			const aPOL = arrPlusOne.length;
+
+			expect(aPOL).toEqual(6);
+			expect(arrPlusOne[aPOL - 1]).toEqual(5);
```
Comment (bradencodes) [inline basic-javascript/project-2.spec.js:328]: It would be better to use an array for the expected value instead of just testing the length of the array.

File: basic-javascript/project-2.spec.js [hunk 5]
```diff
@@ -1,3 +1,519 @@
...
+		// 		sumOfNumbers += numbers[i];
+		// 	}
+		// 	return sumOfNumbers;
+		// };
+		// expected sum
+		it('should return the sum of all numbers in the array', () => {
+			const numbers = [0, 1, 2, 3, 4];
+			const numbersPlus = [0, 1, 2, '4', 3];
+			const test1 = funcs.addNumbers(numbers);
+			const test2 = funcs.addNumbers(numbersPlus);
```
Comment (bradencodes) [inline basic-javascript/project-2.spec.js:446]: Interesting test with throwing a string into the array along with numbers. That's good for understanding exactly what the code does.

**Comment by bradencodes** [comment]
## MVP
- [x] Write tests for project-1.js
- [x] Write tests for project-2.js

## Stretch
- [x] Correct bugs and refactor the code
- [ ] Write tests for arrays.js
- [ ] Write tests for object.js

## What Looks Great
- MVP complete! Nice work
- Fixed some bugs in the code
- Frequent and descriptive commits
- Code is well formatted and easy to read
- Titles for tests are very descriptive
- Tests are thorough and check edge cases
- Included functions as comments in each test

## What Could Be Added
- Write tests for stretch for even more practice!

## Rating (1-3)
**3**

File: basic-javascript/project-2.spec.js (hunk 4 above)
Comment (VeraSimon) [reply basic-javascript/project-2.spec.js:328]: @Bradiowave I'm doing both. What do you mean?
